_includes/tutorials/kafka-connect-datagen/confluent/markup/dev/provision-connector.adoc

```diff
@@ -1,23 +1,23 @@
+*Option 1.* You can use the Confluent Cloud CLI which provides the `ccloud connector create` command allowing you to pass in the configuration file from the previous step.
 
 +++++
-<pre class="snippet"><code class="shell">{% include_raw tutorials/kafka-connect-datagen-ccloud/kafka/code/tutorial-steps/dev/create-connector.sh %}</code></pre>
+<pre class="snippet"><code class="shell">{% include_raw tutorials/kafka-connect-datagen/confluent/code/tutorial-steps/dev/create-connector.sh %}</code></pre>
 +++++
 
-*Option 2.* The Confluent Cloud REST API can provision the connector using the configuration file you created from the previous step. This API requires we provide a Confluent Cloud resource ID for both the environment and Kafka cluster we wish to deploy the connector to. These values can be obtained by using the link:https://confluent.cloud/environments[Confluent Cloud UI] or using the `ccloud kafka cluster describe` link:https://docs.confluent.io/ccloud-cli/current/command-reference/kafka/cluster/ccloud_kafka_cluster_describe.html[command]. When using `ccloud_library.sh`, as we did above, the script sets two environment variables (`ENVIRONMENT` and `CLUSTER`) with these values.
+*Option 2.* The Confluent Cloud REST API can provision the connector using the configuration file you created from the previous step. This API requires we provide a Confluent Cloud resource ID for both the environment and Kafka cluster we wish to deploy the connector to. These values can be obtained by using the link:https://confluent.cloud/environments[Confluent Cloud UI] or using the `ccloud kafka cluster describe` link:https://docs.confluent.io/ccloud-cli/current/command-reference/kafka/cluster/ccloud_kafka_cluster_describe.html[command].
 
 Additionally, we must provide an API Key and Secret which authorizes us to control our cloud account. This API key is independent of the one you use to connect to Kafka or the Schema Registry, so we need to generate it before the HTTP command will succeed.
 
 +++++
-<pre class="snippet"><code class="shell">{% include_raw tutorials/kafka-connect-datagen-ccloud/kafka/code/tutorial-steps/dev/create-cloud-api-key.sh %}</code></pre>
+<pre class="snippet"><code class="shell">{% include_raw tutorials/kafka-connect-datagen/confluent/code/tutorial-steps/dev/create-cloud-api-key.sh %}</code></pre>
 +++++
 
 The `cloud-api-key.json` file now contains an API key and secret authorized to control your cloud account. Protect this file as you would any secret value. 
 
 The following `curl` command will read the API key and secret from the `cloud-api-key.json` file (using the link:https://stedolan.github.io/jq/[jq dev tool]) and `PUT` the new connector config to the REST API in the appropriate environment and Kafka cluster.
```

Comment:
IMO we need to make the reader aware of the variables ENVIRONMENT and CLUSTER in this next command. You can't see it be default (because of scroll). You mention the need for them above, but I think we should mention where they need to be substitued in the next step.

Reply:
Good point @rspurgeon .  It's even worse though, now that `ccloud-stack` is removed, user needs to chase down those variables themselves.  I will address

Reply:
@rspurgeon Addressed in latest commit